lecture_05.py

```diff
@@ -1,14 +1,17 @@
-"""Lecture 5 DTN"""
+"""Lecture 5 DTN
+This is a new fun thing of happiness.
+"""
 
 
 def main():
-    names = ["Alice", "Bob", "Carly", "David"]
-    ages = [3, 20, 7, 1]
+    names = ["Alice", "Bob", "Carly", "David", "Lindsay"]
+    ages = [3, 20, 7, 1, 21]
     print(determine_oldest(names, ages))
 
 
 def determine_oldest(names, ages):
     return names[ages.index(max(ages))]
```

Comment:
you could create a variable for the oldest age so it can be used again in another place
eg. oldest_age = ages.index(max(ages))